<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WKWIN5154091" ignored="0" name="Test" tests="5" failures="3" timestamp="31 Jul 2019 17:54:22 GMT" time="0.087" errors="0">
  <testcase name="test1" time="0.019" classname="learnRetryLogicRunTime.TestCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at learnRetryLogicRunTime.TestCases.test1(TestCases.java:13)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test2" time="0.003" classname="learnRetryLogicRunTime.TestCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at learnRetryLogicRunTime.TestCases.test2(TestCases.java:19)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test3" time="0.001" classname="learnRetryLogicRunTime.TestCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at learnRetryLogicRunTime.TestCases.test3(TestCases.java:25)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test4" time="0.003" classname="learnRetryLogicRunTime.TestCases"/>
  <testcase name="test5" time="0.001" classname="learnRetryLogicRunTime.TestCases"/>
</testsuite> <!-- Test -->
